README.md

```diff
@@ -10,3 +9,33 @@ For detailed API description, please see [the draft on the project
 wiki](https://github.com/codingteam/Gestalt/wiki/Public-API).
 
 For copying terms, see LICENSE file.
+
+# Buildig and running the service
+
+You need the following installed in your system:
+
+* [Erlang version 17 or above](erlang.org)
+* [rebar](https://github.com/rebar/rebar/wiki/Getting-started#first-steps)
+
+Now run:
+
+    # compile the source and generate the release
+    $ rebar compile generate
+
+    # start the service
+    $ ./rel/gestalt/bin/gestalt start
+
+You should now see Gestalt's public API responding at port 8080. You can also
+check if the service is alive by pinging it:
+
+    $ ./rel/gestalt/bin/gestalt ping
+
+It's also possible to get a shell attached to the service:
+
+    $ ./rel/gestalt/bin/gestalt attach
+
+(Note: you can quit by pressing ^D.)
+
+To stop the service, run:
+
+    $ ./rel/gestalt/bin/gestalt stop
```

Comment:
[Told you so!](https://github.com/codingteam/Gestalt/issues/9#issuecomment-94174105)